test/features/cloudnative/init_containers.go

```diff
@@ -45,19 +49,21 @@ func checkInitContainers(sampleApp *sample.App) features.Func {
 
 			var oneAgentInstallContainer *corev1.Container
 
+			var initContainerName string
 			for _, initContainer := range podItem.Spec.InitContainers {
-				if initContainer.Name == webhook.InstallContainerName {
+				if initContainer.Name == oldInstallContainerName || initContainer.Name == webhook.InstallContainerName {
 					oneAgentInstallContainer = &initContainer // loop breaks after assignment, memory aliasing is not a problem
+					initContainerName = initContainer.Name
 
 					break
 				}
 			}
-			require.NotNil(t, oneAgentInstallContainer, "'%s' pod - '%s' container not found", podItem.Name, webhook.InstallContainerName)
+			require.NotNil(t, oneAgentInstallContainer, "'%s' pod - '%s' container not found", podItem.Name)
```

Comment:
message needs 2 args

Reply:
I fixed it, refresh the page please

Reply:
https://github.com/Dynatrace/dynatrace-operator/pull/4046/commits/91a584ac2a08884bcb6cebe58af89165624e04c8